admin/class-keyword-synonyms-modal.php

```diff
@@ -19,21 +19,20 @@ class WPSEO_Keyword_Synonyms_Modal {
 	public function get_translations() {
 		return array(
 			'title'                    => __( 'Would you like to add keyword synonyms?', 'wordpress-seo' ),
-			'intro'                    => sprintf(
+			'intro'                    =>
 				/* translators: %1$s expands to a 'Yoast SEO Premium' text linked to the yoast.com website. */
-				__( 'Great news: you can, with %1$s!', 'wordpress-seo' ),
-				'{{link}}Yoast SEO Premium{{/link}}'
-			),
+				sprintf(
```

Comment:
This change is breaking coding standards and needs to be reverted.
This applies to all `translator` comments which are not on the line above `__(`